lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/impl/DefaultRemoteManagerImplementation.java

```diff
@@ -83,11 +85,11 @@
       final LocalAddressProvider localAddressProvider,
       final TransportFactory tpFactory) {
       this(name, hostAddress, listeningPort, codec, errorHandler, orderingGuarantee, numberOfTries, retryTimeout,
-              localAddressProvider, tpFactory, RangeTcpPortProvider.Default);
+              localAddressProvider, tpFactory, getTcpProvider());
```

Comment:
Thank you, @markusweimer . By the way, do you mean to include the deletion of the deprecated `DefaultRemoteManagerImplementation` constructors?